cartographer/io/hybrid_grid_points_processor.h

```diff
@@ -0,0 +1,54 @@
+#ifndef CARTOGRAPHER_IO_HYBRID_GRID_POINTS_PROCESSOR_H_
+#define CARTOGRAPHER_IO_HYBRID_GRID_POINTS_PROCESSOR_H_
+
+// Library used for processing points processor pipeline.
+
+#include <memory>
+#include <string>
+
+#include "cartographer/io/file_writer.h"
+#include "cartographer/io/points_batch.h"
+#include "cartographer/io/points_processor.h"
+#include "cartographer/mapping_3d/hybrid_grid.h"
+#include "cartographer/mapping_3d/range_data_inserter.h"
+
+namespace cartographer {
+namespace io {
+
+// Creates a geolocated hybrid grid of the points with voxels being
```

Comment:
I would like to save a geolocated hybrid grid.  That is a geodetic pose {{lat, lng, alt}, {roll, pitch, yaw}} for the center of the hybrid grid to place it on the surface of the Earth.  The cartographer.mapping_3d.proto.HybridGrid message does not have fields to encode this information.  I proposed a GeolocatedOctree proto that does encode this information.  Perhaps we could wrap the HybridGrid proto with a GeolocatedHybridGrid proto to place it on the surface of the Earth?